indispensable-asn1/src/commonMain/kotlin/at/asitplus/signum/indispensable/asn1/Asn1Exception.kt

```diff
@@ -13,7 +13,9 @@ class Asn1TagMismatchException(val expected: Asn1Element.Tag, val actual: Asn1El
 
 class Asn1StructuralException(message: String, cause: Throwable? = null) : Asn1Exception(message, cause)
 
-class Asn1OidException(message: String, val oid: ObjectIdentifier) : Asn1Exception(message)
+class Asn1OidException(message: String, val oid: ObjectIdentifier) : Asn1Exception(message){
+    constructor(oid: ObjectIdentifier):this("Unknown oid $oid", oid)
```

Comment:
should this be an `UnknownOidException` or similar?

Reply:
since it has two different constructors you can spefify anything that has anything todo with an oid. could be mismatch, whatever. 

I don't think we need overly specific exceptions. since this is mainly useful for debugging and here the message suffices

Reply:
then i'd prefer `internal fun Unknown(oid)` on the companion to make the usage clear in ode